doc/code_style/mom.h

```diff
@@ -113,16 +113,16 @@ class YourMom : public MassiveEntity {
 enum class awesome_enumeration {
 	LOW,
 	MID,
-	HIGH,  // <- note the last , there!
+	HIGH, // <- note the last , there!
 };
 
 
 /**
  * Type aliases are done with "using".
  * They can get proper documentation, too!
  */
-using wtf_t = std::unordered_map<std::string,
-                                 std::unique_ptr<Stuff>>;
+using wtf_t = std::unordered_map<std::string, std::unique_ptr<Stuff>>;
```

Comment:
clang's style is ok, but we do allow and want to showcase template argument wrapping here.

Reply:
Works out, pressing enter after the comma aligns it with the templates `<`. I think this is because the column limit of `100` probably would chose the first style if lower. Had to manually edit that, fyi.